### DISCUSSION TITLE
Configure base URL to redirect

### DISCUSSION THREAD

**Discussion by yaelKeren** [google/docsy#1412] [Q&A]
Hi,
We have a site using docsy theme we want the base URL to redirect to the base.URL/docs.
Can we do that with docsy theme?

**Comment by yaelKeren** [comment]
For example like that:
https://www.docsy.dev ----> https://www.docsy.dev/docs/

**Comment by deining** [comment]
Looks like you want to get rid of all sections except for the `dos` section, correct?
If so, have a look at the `mostlydocs` repo:

https://github.com/gwatts/mostlydocs

The README.md explains the magic that does the trick of displaying the docs only.

Is this of help for you?
